src/EditorFeatures/TestUtilities/Formatting/CoreFormatterTestsBase.cs

```diff
@@ -77,5 +87,206 @@ protected void TestIndentation(TestWorkspace workspace, int indentationLine, int
 
             Assert.Equal(expectedIndentation, actualIndentation);
         }
+
+        protected void AssertFormatWithView(string expectedWithMarker, string codeWithMarker, params (PerLanguageOption<bool> option, bool enabled)[] options)
```

Comment:
these are just moving up from C# to a shared location so we can use this both from C# and VB.